source/reference/disaster-recovery.txt

```diff
@@ -55,6 +55,8 @@ following changes:
        If synchronization was started with write blocking, disable the Write
        Block Mode.
 
+   * - :ref:`Capped Collections <c2c-dr-capped-collections>`
```

Comment:
I don't really want them to say the same thing. This should be a summary compared to below which has all of the information.

What about putting something like: "Synchronization sets capped collections to the maximum allowable size. Use the collMod command to resize capped collections"